vxsandbox/resources/tests/test_metrics.py

```diff
@@ -98,6 +98,10 @@ def setUp(self):
         super(TestMetricsResource, self).setUp()
         yield self.create_resource({})
 
+    # def create_resource(self, config):
+    #     config.setdefault('metrics_prefix', {})
+    #     return super(TestMetricsResource, self).create_resource(config)
```

Comment:
We'll need to set the `metrics_prefix` in the test config in order to get the tests to pass again.